frontend/scss/components/templates/roadmap.scss

```diff
@@ -228,31 +237,47 @@
       width: 80%;
     }
 
-    &-quarter-separator h3 {
-      font-size: 28px;
-      opacity: 0.2;
-      margin-bottom: 0px;
-
-      @media (min-width: 768px) {
-        display: none;
-      }
-    }
-
     &-item {
       position: relative;
       padding-top: 20px;
 
+      &:first-child {
+        padding-top: 0;
+      }
+
       &:last-child {
         #{$root}-list-item-timeline--vertical {
```

Comment:
Is this ever shown? Otherwise why not not render it in the template via `loop.last` in the first place?